.travis.yml

```diff
@@ -58,19 +64,28 @@ matrix:
             - fonts-liberation
       env:
         - JOB=stability SCRIPT=tools/ci/ci_stability.sh PRODUCT=chrome:dev
-    - python: 2.7
+    - name: "tools unittests (Python 2)"
+      os: linux
+      python: "2.7"
       env: JOB=tools_unittest TOXENV=py27 HYPOTHESIS_PROFILE=ci SCRIPT=tools/ci/ci_tools_unittest.sh
-    - python: 3.6
+    - name: "tools unittests (Python 3)"
+      os: linux
+      python: "3.6"
       env: JOB=tools_unittest TOXENV=py36 HYPOTHESIS_PROFILE=ci SCRIPT=tools/ci/ci_tools_unittest.sh
-    - python: 2.7
+    - name: "resources unittests"
+      os: linux
+      python: "2.7"
       env: JOB=resources_unittest TOXENV=py27 SCRIPT=tools/ci/ci_resources_unittest.sh
-    - python: 2.7
+    - name: "flake8"
+      os: linux
+      python: "2.7"
       addons:
         apt:
           packages:
             - libnss3-tools
       env: JOB=wpt_integration TOXENV=py27,py27-flake8 SCRIPT=tools/ci/ci_wpt.sh
-    - os: linux
+    - name: "wptrunner infrastructure"
```

Comment:
Is this a good name? Are these unit tests, or what?

Reply:
This runs pytest in `tools/wpt`

Reply:
Hmm, that doesn't seem quite right. The only place pytest appears in tools/ci is in tools/ci/tests/test_jobs.py, and this job will run tools/ci/ci_wptrunner_infrastructure.sh. Looks like it will run infrastructure/ in chrome and firefox, using in-tree expectations. Am I reading that right?

If so, maybe call it "infrastructure/ tests"? I'll throw on a slash on some of the other names too for consistency.